### DISCUSSION TITLE
Accessing generated MPEG-DASH files

### DISCUSSION THREAD

**Discussion by bandirsen** [bluenviron/mediamtx#1253] [General]
I was successfully generated dash file using this configuration
```
paths:
  dash:
    runOnPublish: ffmpeg -re -i rtsp://localhost:8554/$RTSP_SERVER_PATH -c copy -f dash dash/stream.mpd
```

But I can't find a way for dash player to access that generated MPEG-DASH files, I've been trying 
http://rtsp.simple.server.ip:8888/streamname/dash/stream.mpd
http://rtsp.simple.server.ip:8888/dash/stream.mpd
http://rtsp.simple.server.ip:8888/stream.mpd

but none working or should i run separated file server to serve those generated MPEG-DASH files ?

**Comment by aler9** [comment]
> should i run separated file server to serve those generated MPEG-DASH files ?

yes :)

DASH support was never integrated into the server since HLS is currently supported by most platforms.

**Reply by bandirsen** [reply]
Sorry for late response,
Thanks for your explanation
Regards

[a comment by an automated account is not shown]
